lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -168,14 +169,13 @@ window.runLighthouseInWorker = function(port, url, options, categoryIDs) {
  * @param {!string} reportContext Where the report is going
  * @return {!string} Blob URL of the report (or error page) HTML
  */
-window.createReportPageAsBlob = function(results, reportContext) {
+window.createReportPageAsBlob = function(results) {
   performance.mark('report-start');
-  const reportGenerator = new ReportGenerator();
   let html;
   try {
-    html = reportGenerator.generateHTML(results, reportContext);
+    html = new ReportGeneratorV2().generateReportHtml(results);
   } catch (err) {
-    html = reportGenerator.renderException(err, results);
+    html = new ReportGenerator().renderException(err, results);
```

Comment:
can probably ditch this too (like in the CLI) since an exception in v2 generation is extremely unlikely